### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by the-andrew-007**

### CODE REVIEW

File: services/movie_session.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import datetime
+
+from db.models import MovieSession
+from django.db.models.query import QuerySet
+
+
+# from datetime import time, date
```
Comment (katevasilenko) [inline services/movie_session.py:7]: Please remove

File: services/movie_session.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+    session_id: int,
+    show_time: datetime.time = None,
+    movie_id: int = None,
+    cinema_hall_id: int = None,
+) -> None:
+    params = {
```
Comment (katevasilenko) [inline services/movie_session.py:40]: Not really good naming. Please provide more clear name of variable

File: services/movie_session.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        "show_time": show_time,
+        "movie_id": movie_id,
+        "cinema_hall_id": cinema_hall_id,
+    }
+    not_none_params = {k: v for k, v in params.items() if v is not None}
```
Comment (katevasilenko) [inline services/movie_session.py:45]: The same

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

**Review by Vasyl-Smutok — APPROVED** [review]
Good Job 👍

**Pull request closed without merging**